lib_comfyui/parallel_utils.py

```diff
@@ -27,6 +27,7 @@ def __init__(self, producer, *args, ctx=None, **kwargs):
         super(SynchronizingQueue, self).__init__(*args, ctx=ctx, **kwargs)
         self._consumer_ready_event = multiprocessing.Event()
         self._producer = producer
+        self.state_dict_thread = None
```

Comment:
I'm really not convinced `self.state_dict_thread` goes in here. Should we create another class instead? It just seems weird that a queue has a thread.

Reply:
I agree that this should be another class